### PR TITLE
Continue migrating #135 (letter 'b')

### PR DESCRIPTION
**Pull request by luzpaz**
This is all entries of the letter 'b'

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -991,7 +1003,11 @@ begginings->beginnings
 beggins->begins
 begining->beginning
 beginnig->beginning
-behaivior->behaviour
+behaivior->behavior, behaviour,
+behaviour->behavior, behaviour,
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: `behaviour`is british spelling. Do we want to have this? Or should we set up a flag to include british spelled words ?

Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: I'd rather leave handling of British spellings to a different discussion/PR than one that aims to simply add a lot of words

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: The dictionary that I'm converting orients to british spelling. So how do you wish I proceed. Should we just add everything and then sort it out after the migration ?

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: > I'd rather leave handling of British spellings to a different discussion/PR than one that aims to simply add a lot of words

Makes sense. I'm just not sure how to proceed. Should I: 
1) take out all the British spelling typos? Or  
2) take out both British typos and solutions?  
Ex.  
1) Remove just `behaviour->behavior, behaviour,` Or  
2) Commit the following instead:    
behaivior->behavior, ~behaviour~,
~behaviour->behavior, behaviour~,
behavoir->behavior, ~behaviour~,
behavour->behavior, ~behaviour~,
behavoirs->behaviors, ~behaviours~,

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: I'll just remove British words altogether and add them to #103

**Comment by luzpaz** [comment]
All British spelling has been purged. Please review.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: @larsoner ping

**Comment by luzpaz** [comment]
@lucasdemarchi care to merge this?

**Comment by luzpaz** [comment]
@larsoner what holding this back from merge? please advise

**Comment by luzpaz** [comment]
please advise... not sure why we're delaying merging...

**Comment by lucasdemarchi** [comment]
Just lack of time to review, sorry.

**Comment by luzpaz** [comment]
@pabs3 care to review and merge ?

**Pull request merged**